.github/workflows/ci-tests.yml

```diff
@@ -91,10 +100,10 @@ jobs:
       - name: "conda environment cache"
         uses: ./.github/workflows/composite/conda-env-cache
         with:
-          cache_build: 0
+          cache_build: 1
           cache_period: ${{ env.CACHE_PERIOD }}
           env_name: ${{ env.ENV_NAME }}
-          install_packages: "cartopy nox pip"
+          install_packages: "cartopy tox pip"
```

Comment:
@lbdreyer I think this should be `tox<4` atm as `tox-conda` isn't 4.x safe. Also `tox-wheel` hasn't been released since Oct 2022, so it's probably not 4.x compliant either, you'd have to check.

Note that `tox` 4.x was released in Dec 2022.